cmd/mr_create.go

```diff
@@ -121,10 +121,13 @@ func runMRCreate(cmd *cobra.Command, args []string) {
 
 	if sourceTarget != "" {
 		sourceParts := strings.Split(sourceTarget, ":")
+		if len(sourceParts) < 2 {
+			log.Fatalf("source remote must have format remote:remote_branch")
+		}
 		sourceRemote = sourceParts[0]
 		sourceBranch = sourceParts[1]
 		if sourceRemote == "" || sourceBranch == "" {
-			log.Fatalf("source remote must have format remote:remote_branch.\n")
+			log.Fatalf("source remote must have format remote:remote_branch")
 		}
```

Comment:
Is this check still valid? AFIK the problems that should've been catched here, are now being treated by lines 124:126, right?

Reply:
It's still valid: if the user uses `--source whatever:` the `strings.Split(sourceTarget, ":")` returns a list of length of 2, being that the first value is `whatever` and the second one is an empty string. So we still need to make sure the user explicitly passed both `sourceRemote` and `sourceBranch`.

Reply:
But yeah, I agree. Seems odd to have two similar checks one after the other. I spent some minutes trying to figure out if I was missing something.

Reply:
Got it, I was misunderstanding the Split(), now it makes sense to me.

On the other hand, way to eliminate the one check would be to:
`if len(sourceParts) < 2  || sourceParts[0] == "" || sourceParts[1] == "" { `
But hey, I'm fine with how it is :)

Reply:
Yeah, that's right. Actually your suggestion gave me another thing to check.
I'm going to send a new version of the patch trimming all whitespaces.